rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/mqtt/TbMqttNode.java

```diff
@@ -154,7 +159,7 @@ protected void prepareMqttClientConfig(MqttClientConfig config) throws SSLExcept
         }
     }
 
-    private SslContext getSslContext() throws SSLException {
+    protected SslContext getSslContext() throws SSLException {
```

Comment:
should be private. Use verify on method: prepareMqttClientConfig to check the sslContext.